Tests/SwiftBoxMetricsTests/HandlersTests.swift

```diff
@@ -81,7 +75,9 @@ class StatsDHandlerTests: XCTestCase {
 
     static var allTests: [(String, (StatsDHandlerTests) -> () throws -> Void)] {
         return [
-            ("testHandlerShouldGatherMetricsProperly", testHandlerShouldGatherMetricsProperly),
+            ("testHandlerShouldGatherTimerMetricsProperly", testHandlerShouldGatherTimerMetricsProperly),
+            ("testHandlerShouldGatherRecorderMetricsProperly", testHandlerShouldGatherRecorderMetricsProperly),
```

Comment:
Unit test names should follow `testXShouldYWhenZ`. If we don't have Z scenario, we should get rid of "propertly" suffix, because it doesn't mean anything.